README.md

```diff
@@ -32,18 +38,66 @@ python setup.py sdist
 sudo pip install --upgrade dist/CommentSearchEngine-0.1.0.tar.gz
```

Comment:
mark this as optional since you don't need to actually do it

Reply:
see f8c3e55